packages/app/src/debug/DebugArtifactLink.cy.tsx

```diff
@@ -0,0 +1,40 @@
+import DebugArtifactLink from './DebugArtifactLink.vue'
+
+describe('<DebugArtifacts />', () => {
+  const artifactMapping: {icon: string, text: string, url: string}[] = [
+    { icon: 'TERMINAL_LOG', text: 'View Log', url: 'www.cypress.io' },
+    { icon: 'IMAGE_SCREENSHOT', text: 'View Screenshot', url: 'cloud.cypress.io' },
+    { icon: 'PLAY', text: 'View Video', url: 'www.cypress.io' },
+  ]
+
+  it('mounts correctly, provides expected tooltip content, and emits correct event', () => {
+    artifactMapping.forEach((artifact) => {
+      cy.mount(() => (
+        <DebugArtifactLink icon={artifact.icon} popperText={artifact.text} url={artifact.url}/>
+      ))
+
+      cy.findByTestId(`artifact-for-${artifact.icon}`).should('have.length', 1)
+      cy.findByTestId(`${artifact.icon}-button`).should('be.visible')
+      cy.findByTestId(`artifact-for-${artifact.icon}`).realHover().then(() => {
+        cy.findByTestId('tooltip-content').should('be.visible').contains(artifact.text)
+      })
+
+      cy.percySnapshot()
+    })
+  })
+
+  it('mounts correctly for all icons together and has correct URLs', () => {
+    cy.mount(() => (
+      <div class="flex flex-grow space-x-4.5 pt-24px justify-center" data-cy='debug-artifacts-all'>
```

Comment:
Noticed an opportunity here, we could grab the very similar looking code [here](https://github.com/cypress-io/cypress/blob/ankit/IATR-M1.0-spec-component-update/packages/app/src/debug/DebugFailedTest.vue#L36-L53) and [here](https://github.com/cypress-io/cypress/blob/ankit/IATR-M1.0-spec-component-update/packages/app/src/debug/GroupedDebugFailedTest.vue#L17-L32) to make a `DebugArtifactLinkList` component or something. We seem to have the idea of this component 3 places if you include this test, and the presence in the test tells me we want to see the artifact links grouped on their own like this.